.travis.yml

```diff
@@ -16,3 +22,4 @@ addons:
 
 env:
   - CXX=g++-4.8
+  - DB_HOST="127.0.0.1" DB_PORT=3306 DB_USER="bitstarterTest" DB_PASS="bitstarterTest" DB_NAME="bitstarterTest"
```

Comment:
Each line under the `env:` section will start a new test environment in travis. Each of those environments will run all the tests. You must combine the environment variables here into one line, like this:
```yml
env:
   - CXX=g++-4.8 DB_HOST="127.0.0.1" DB_PORT=3306 DB_USER="bitstarterTest" DB_PASS="bitstarterTest" DB_NAME="bitstarterTest"
```